components/layout/DocsLayout.tsx

```diff
@@ -86,50 +89,76 @@ export default function DocsLayout({ post, navItems = {}, children }: IDocsLayou
 
   const navigation = posts.docsTree;
 
+  const sidebar = (
```

Comment:
Can we have a separate component for the DocsSidebar then? It will be then better maintained in future UI tests, instead of creating multiple components in same file.

Reply:
Nice suggestion @akshatnema, I'll do that :)